FlowCrypt/Controllers/CheckMailAuth/CheckMailAuthViewController.swift

```diff
@@ -13,8 +13,10 @@ import UIKit
 
 class CheckMailAuthViewController: TableNodeViewController {
     private let globalRouter: GlobalRouterType
+    private let appContext: AppContext
 
-    init(globalRouter: GlobalRouterType = GlobalRouter()) {
+    init(appContext: AppContext, globalRouter: GlobalRouterType = GlobalRouter()) {
+        self.appContext = appContext
         self.globalRouter = globalRouter
         super.init(node: TableNode())
     }
```

Comment:
Context now gets passed as first argument of every VC init.

Next step will be to pass `User` (not optional) into every VC that requires it. #863